ios/wrappers/utils/CioConstants.swift

```diff
@@ -14,28 +14,17 @@ struct CustomerioConstants {
     // Push Messaging
     static let CioDeliveryId = "CIO-Delivery-ID"
     static let CioDeliveryToken = "CIO-Delivery-Token"
-    
-    // Tracking
-    static let siteId = "siteId"
-    static let apiKey = "apiKey"
-    static let region = "region"
-    static let version = "version"
-    static let source = "source"
-    static let expo = "expo"
-    
-    static let autoTrackDeviceAttributes = "autoTrackDeviceAttributes"
-    static let logLevel = "logLevel"
-    static let autoTrackPushEvents = "autoTrackPushEvents"
-    static let bgQMinTasks = "backgroundQueueMinNumberOfTasks"
-    static let bgQSecondsDelay = "backgroundQueueSecondsDelay"
-    static let trackingApiUrl = "trackingApiUrl"
-    static let enableInApp = "enableInApp"
-    
-    static let cioTag = "[CIO]"
     static let showPromptFailureError = "Error requesting push notification permission."
     static let showDeviceTokenFailureError = "Error fetching registered device token."
     static let platformiOS = "ios"
     static let sound = "sound"
     static let badge = "badge"
     
+    // Logging
+    static let cioTag = "[CIO]"
+    
+    // DataPipelines
+    static let initializeWithConfigMessage = "Initializing Customer.io SDK with config:"
+    static let noIdOrTraitError = "Provide id or traits to identify a user profile."
+    static let initializationFailedError = "Initializing Customer.io SDK failed with error"
```

Comment:
they don't seem to be repeated, i wonder whats the benefit behind making them constants? specially when they are just used for logging and one time only?

Reply:
They are currently not repeated but I used constants because the code looks clean and readable with them.